src/IssueViz.Security/HotspotsList/TableDataSource/HotspotsTableDataSource.cs

```diff
@@ -33,6 +33,7 @@ namespace SonarLint.VisualStudio.IssueVisualization.Security.HotspotsList.TableD
     internal interface IHotspotsStore
     {
         void Add(IAnalysisIssueVisualization hotspot);
+        void Remove(IAnalysisIssueVisualization hotspot);
```

Comment:
The interface method is not used externally, but I think it still makes sense to have it (and soon we will probably add "RemoveAll()" for #1801)